source/tutorials/nixos/nixos-configuration-on-vm.md

```diff
@@ -180,13 +180,19 @@ ls -R ./result
     run-nixos-vm
 ```
 
-Run the virtual machine:
+To run the virtual machine in a new window (requires GUI environment):
```

Comment:
```suggestion
Run the virtual machine in the current terminal:
```

Let's not show all the options here, that's what reference documentation is for. The purpose of this tutorial (as any tutorial here) is to serve as a first encounter, with pointers to all the details.

I prefer the terminal version as that is or at least appears more lightweight, and showing a the flag here should provide enough of a hint that omitting it may give you a GUI.